typescript/ws-protocol/README.md

```diff
@@ -1,43 +1,32 @@
-# Foxglove Studio WebSocket protocol
+# Foxglove WebSocket protocol in TypeScript
 
-This package provides server and client implementations of the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
-
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The Foxglove WebSocket protocol is encoding-agnostic, as long as the desired encoding is supported by both client and server. It currently supports Protobuf and JSON messages.
 
 ## Installation
 
+Install the `ws-protocol` `npm` package:
```

Comment:
The name is `@foxglove/ws-protocol`, not just `ws-protocol`. I also find `npm` strange, we can probably just say npm.

On the other hand, overall I don't think this sentence is necessary because the line of code below basically says the exact same thing.